app/src/main/java/com/hv/bukutm/presentation/screen/dashboard/penerimatamu/ScanQrScreen.kt

```
@@ -522,19 +678,399 @@ private fun AppointmentDetailCard(
+    var timeError by remember { mutableStateOf<String?>(null) }
+    var showDatePicker by remember { mutableStateOf(false) }
+    var showTimePicker by remember { mutableStateOf(false) }
+    var dateDialogError by remember { mutableStateOf<String?>(null) }
+    var timeDialogError by remember { mutableStateOf<String?>(null) }
+
+    val today = LocalDate.now()
+    val maxDate = today.plusMonths(1)
+    val currentTime = LocalTime.now()
+    val isSelectedDateToday = try {
+        LocalDate.parse(date, DateTimeFormatter.ofPattern("yyyy-MM-dd")).isEqual(today)
+    } catch (e: Exception) { false }
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)